src/result_generation/height/height.py

```diff
@@ -47,17 +50,38 @@ def __init__(
                 self.result_generation.scan_metadata['id'] / self.rgb_input_format
         self.artifact_workflow_obj['id'] = self.result_generation.workflows.get_workflow_id(
             self.artifact_workflow_obj['name'], self.artifact_workflow_obj['version'])
+        self.standing_laying_workflow_obj['id'] = self.result_generation.workflows.get_workflow_id(
+            self.standing_laying_workflow_obj['name'], self.standing_laying_workflow_obj['version'])
         self.scan_workflow_obj['id'] = self.result_generation.workflows.get_workflow_id(
             self.scan_workflow_obj['name'], self.scan_workflow_obj['version'])
 
+    def get_standing_results(self):
+        url = os.getenv('APP_URL', 'http://localhost:5001')
+        cgm_api = ApiEndpoints(url)
+        result = cgm_api.get_results(self.standing_laying_workflow_obj['id'], self.result_generation.scan_metadata['id'])
+        artifact_id_dict_by_order_id = {}
+        sl_data_dict_by_order_id = {}
+        for image_artifact in self.image_artifacts:
+            artifact_id_dict_by_order_id[image_artifact['id']] = image_artifact['order']
+        for r in result:
+            rgb_image_id = r['source_artifacts'][0]
+            sl_data_dict_by_order_id[artifact_id_dict_by_order_id[rgb_image_id]] = float(r['data']['standing_laying'][1:-1])
```

Comment:
There may be the case that key(rgb_image_id) is not present in artifact_id_dict_by_order_id.
Handle above scenario